documentation/index.markdown

```diff
@@ -26,6 +26,8 @@ Puppet Server is the next-generation application for managing Puppet agents.
 * **Administrative API**
     * [Environment Cache](./admin-api/v1/environment-cache.markdown)
     * [JRuby Pool](./admin-api/v1/jruby-pool.markdown)
+* **Puppet API**
+    * [Environment Classes](./puppet/v3/environment_classes.markdown)
```

Comment:
@gguillotte I have no idea how you want all of the links for this stuff to look.  I figured you might have a strategy in mind already for how the static content APIs are going to be covered for Puppet Server that we'd want to follow here.  I'm happy to pull whatever links out of these docs that you'd like that might otherwise cause you more grief later.